.circleci/push_doc.sh

```diff
@@ -62,9 +62,38 @@ touch .nojekyll
 # landing page on fairlearn.org
 if [ "$CIRCLE_BRANCH" = "main" ]
 then
-    echo "Copying landing_page.js into js directory"
+    echo "Copying all the newly generated  files for the static landing page"
     ls $GENERATED_DOC_DIR/..
+    #js
     cp $GENERATED_DOC_DIR/../landing_page.js js/
+    #css
+    cp $GENERATED_DOC_DIR/../landing_page_style.css css/
+    #html
+    cp $GENERATED_DOC_DIR/../index.html  .
+    #fonts
+    cp $GENERATED_DOC_DIR/../SourceCodePro-Bold.ttf fonts/
+    cp $GENERATED_DOC_DIR/../SourceCodePro-Medium.ttf fonts/
+    cp $GENERATED_DOC_DIR/../SourceCodePro-Regular.ttf fonts/
+    #images
+    cp $GENERATED_DOC_DIR/../Button_GH_Default.svg images/
+    cp $GENERATED_DOC_DIR/../Button_GH_Hover.svg images/
+    cp $GENERATED_DOC_DIR/../Discord_Color.svg images/
+    cp $GENERATED_DOC_DIR/../Discord_Inverse.svg images/
+    cp $GENERATED_DOC_DIR/../fairlearn_logo.svg images/
+    cp $GENERATED_DOC_DIR/../fairlearn-favicon.ico images/
+    cp $GENERATED_DOC_DIR/../fairlearn-meta-image.jpg images/
+    cp $GENERATED_DOC_DIR/../Github_Color.svg images/
+    cp $GENERATED_DOC_DIR/../Github_Inverse.svg images/
```

Comment:
can we replace all of these with 
```
cp $GENERATED_DOC_DIR/../*.svg images/
cp $GENERATED_DOC_DIR/../*.ttf images/

```
?

Reply:
Thanks , this looks way more efficient than mine😆